### PR TITLE
37 migrate to create streaming table api from create streaming live table

### PR DESCRIPTION
**Pull request by ravi-databricks**
1. Upgraded dlt api from `create_streaming_live_table` to latest `create_streaming_table`
2. Added data quality expectations for silver since `create_streaming_table` api takes `expectations` as arguments
3. Added unit tests
4. Corrected demo's and integration tests to incorporate silver_data_quality_expectations

### CODE REVIEW

**Review by neil90 — APPROVED** [review]
LGTM

**Review by howardwu-db — APPROVED** [review]
Approved changes

**Review by ganeshchand — APPROVED** [review]
LGTM

**Review by ganeshchand — APPROVED** [review]
LGTM

**Review by ganeshchand — APPROVED** [review]
LGTM

File: integration_tests/run_integration_tests.py [hunk 1]
```diff
@@ -668,7 +668,15 @@ def init_dltmeta_runner_conf(self, runner_conf: DLTMetaRunnerConf):
...
         print(f"uploading to {runner_conf.dbfs_tmp_path}/{self.base_dir}/")
-        self.ws.dbfs.create(path=runner_conf.dbfs_tmp_path + f"/{self.base_dir}/", overwrite=True)
+        if runner_conf.uc_catalog_name:
+            self.ws.dbfs.create(path=runner_conf.dbfs_tmp_path + f"/{self.base_dir}/", overwrite=True)
+        else:
+            try:
+                self.ws.dbfs.mkdirs(runner_conf.dbfs_tmp_path + f"/{self.base_dir}/")
+            except Exception as e:
+                print(f"Error in creating directory {runner_conf.dbfs_tmp_path + f'/{self.base_dir}/'}")
+                print(e)
```
Comment (nfx) [inline integration_tests/run_integration_tests.py:678]: Use logger, not print statements

File: integration_tests/run_integration_tests.py [hunk 2]
```diff
@@ -726,7 +734,7 @@ def run(self, runner_conf: DLTMetaRunnerConf):
             print(e)
         finally:
             print("Cleaning up...")
-            self.clean_up(runner_conf)
+    #        self.clean_up(runner_conf)
```
Comment (nfx) [inline integration_tests/run_integration_tests.py:737]: No commented code

**Review by nfx — APPROVED** [review]

**Pull request merged**